src/makeCalculator.js

```diff
@@ -3,8 +3,51 @@
 /**
  * @return {object}
  */
+
+/* eslint-disable no-console */
+
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      this.result += number;
```

Comment:
Good use of the addition assignment operator `+=` in line with the code style best practices.